network-protocols.md

```diff
@@ -62,6 +62,34 @@ func SayHello(p PeerID) {
 
 Upon receiving a 'hello' stream from another node, you should read off the CBOR RPC message, and then check that the genesis hash matches you genesis hash. If it does not, that node is not part of your network, and should probably be disconnected from. Next, the `HeaviestTipSet`, claimed `HeaviestTipSetHeight`, and peerID of the other node should be passed to the chain sync subsystem.
```

Comment:
I'm super unclear as to why the `HeaviestTipSetHeight` matters. I believe we should only care about the `HeaviestTipSetWeight`.

Reply:
Youre right, weight matters not height